src/config.ts

```diff
@@ -3035,4 +3041,43 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
 		features: ['stargate', 'ibc-transfer', 'no-legacy-stdTx', 'ibc-go'],
 		explorerUrlToTx: 'https://www.mintscan.io/fetchai/txs/{txHash}',
 	},
+	{
+		rpc: 'https://rpc.provenance.io/',
+		rest: 'https://api.provenance.io',
+		chainId: 'pio-mainnet-1',
+		chainName: 'Provenance',
+		stakeCurrency: {
+			coinDenom: 'HASH',
+			coinMinimalDenom: 'nhash',
+			coinDecimals: 9,
+			coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+		},
+		bip44: {
+			coinType: 505,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('pb'),
+		currencies: [
+			{
+				coinDenom: 'HASH',
+				coinMinimalDenom: 'nhash',
+				coinDecimals: 9,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+			},
+		],
+		feeCurrencies: [
+			{
+				coinDenom: 'HASH',
+				coinMinimalDenom: 'nhash',
+				coinDecimals: 9,
+				coinImageUrl: window.location.origin + '/public/assets/tokens/hash.svg',
+			},
+		],
+		gasPriceStep: {
+			low: 1905,
```

Comment:
note, this is in coinDenom (so, HASH), not minimalCoinDenom (nhash). So gas will cost thousands of HASH tokens? I suspect you may mean to put 0.000001905, etc.